post_report.py

```diff
@@ -0,0 +1,86 @@
+
+from pywell.entry_points import run_from_cli
+from pywell.notify_slack import notify_slack
+import requests
+
+from get_pending_accounts import get_pending_accounts
+from filter_emails_by_membership import filter_emails_by_membership
+
+
+DESCRIPTION = 'Post a report on pending accounts from Mobilize.'
+
+ARG_DEFINITIONS = {
+    'DB_HOST': 'Database host IP or hostname',
+    'DB_PORT': 'Database port number',
+    'DB_USER': 'Database user',
+    'DB_PASS': 'Database password',
+    'DB_NAME': 'Database name',
+    'MOBILIZE_API_KEY': 'API key for mobilize.io',
+    'MOBILIZE_API_SECRET': 'API secret for mobilize.io',
+    'MOBILIZE_API_ROOT': 'Root URL for mobilize.io API',
+    'MOBILIZE_DEFAULT_GROUP_ID': 'The first group members request to join',
+    'SLACK_WEBHOOK': 'Web hook URL for Slack.',
+    'SLACK_CHANNEL': 'Slack channel to send to.',
+    'VERBOSE': 'Give verbose output if set'
+}
+
+REQUIRED_ARGS = [
+    'DB_HOST', 'DB_PORT', 'DB_USER', 'DB_PASS', 'DB_NAME',
+    'MOBILIZE_API_KEY', 'MOBILIZE_API_SECRET', 'MOBILIZE_API_ROOT',
+    'MOBILIZE_DEFAULT_GROUP_ID', 'SLACK_WEBHOOK', 'SLACK_CHANNEL'
+]
+
+
+def post_report(args) -> list:
```

Comment:
Can you write a unit test to show we get the output we expect here? [I think this would either require mocking get_pending_accounts somehow or passing in accounts as an argument, but better now while the project is still small...]